### ISSUE TITLE
LineSeries and BarSeries same graph

### ISSUE THREAD

**Issue by dinonovak** [oxyplot/oxyplot#1706]
Is there possibiltiy to show line series and bar series data on the same graph?
X axis should be DateTimeAxis so line and bar should be aligned for each point?

Thanx,
D

**Comment by Jonarw** [comment]
Yes it is possible, however the Bar series requires a `CategoryAxis` as X axis.
You could take a look at the [`BarSeries > Mixed with LineSeries`](https://github.com/oxyplot/oxyplot/blob/ed21516bf69391afd4b37e894d17e50c7d3b209a/Source/Examples/ExampleLibrary/Series/BarSeriesExamples.cs#L348) example in the ExampleBrowser.

**Comment by IAmAgainst** [comment]
Check out the question I just opened related to this but apparently there needs to be an empty ColumnItem for each date in which you don't have a DataPoint in the line series.

![Screenshot_20201108-132327](https://user-images.githubusercontent.com/39332391/98464834-c7d79380-21c5-11eb-816b-0a128f4acb59.png)
